### PR TITLE
Replace outdated references to master with main

### PR DESCRIPTION
**Pull request by Andarist**
cc @elliot-nelson

### CODE REVIEW

File: .changeset/README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 Hello and welcome! This folder has been automatically generated by `@changesets/cli`, a build tool that works with mono-repos or single package repos to help you verion and release your code. You can find the full documentation for it [in our repository](https://github.com/changesets/changesets)
 
-We have a quick list of common questions to get you started engaging with this project in [our documentation](https://github.com/changesets/changesets/blob/master/docs/common-questions.md)
+We have a quick list of common questions to get you started engaging with this project in [our documentation](https://github.com/changesets/changesets/blob/main/docs/common-questions.md)
```
Comment (Andarist) [inline .changeset/README.md:5]: somewhat unrelated to this repo - but Changesets are also using the `main` branch nowadays so I updated that as well

File: .github/workflows/changeset-release.yml [hunk 2]
```diff
@@ -3,22 +3,22 @@ name: Release
...
+      - main
 
 jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@master
+        uses: actions/checkout@v3
```
Comment (Andarist) [inline .github/workflows/changeset-release.yml:14]: those are also a little bit unrelated, but we should at least stick to a specific major version of those actions - so I've replaced those master references here as well

File: .github/workflows/changeset-release.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ jobs:
       - run: git branch -a
 
       - name: "Create Pull Request or Publish to npm"
-        uses: changesets/action@master
+        uses: changesets/action@v3
```
Comment (Andarist) [inline .github/workflows/changeset-release.yml:31]:
```suggestion
        uses: changesets/action@v1
```

**Review by elliot-nelson — APPROVED** [review]

**Comment by elliot-nelson** [comment]
LGTM

**Pull request merged**